common/scala/src/main/scala/whisk/common/Https.scala

```diff
@@ -55,17 +45,17 @@ object Https {
         Some(TLSClientAuth.none)
     }
 
-// configure keystore
     val keystorePassword = httpsConfig.keystorePassword.toCharArray
-    val ks: KeyStore = getCertStore(keystorePassword, httpsConfig.keystoreFlavor, httpsConfig.keystorePath)
+
+    val keyStore: KeyStore = KeyStore.getInstance(httpsConfig.keystoreFlavor)
+    val keyStoreStream: InputStream = new FileInputStream(httpsConfig.keystorePath)
+    keyStore.load(keyStoreStream, keystorePassword)
+
     val keyManagerFactory: KeyManagerFactory = KeyManagerFactory.getInstance(keyFactoryType)
-    keyManagerFactory.init(ks, keystorePassword)
+    keyManagerFactory.init(keyStore, keystorePassword)
 
-// configure truststore
-    val truststorePassword = httpsConfig.truststorePassword.toCharArray
-    val ts: KeyStore = getCertStore(truststorePassword, httpsConfig.truststoreFlavor, httpsConfig.keystorePath)
     val trustManagerFactory: TrustManagerFactory = TrustManagerFactory.getInstance(keyFactoryType)
-    trustManagerFactory.init(ts)
+    trustManagerFactory.init(keyStore)
```

Comment:
As discussed in person, should we add a comment saying that we use keystore as a truststore here or leave truststore configurable from the outside?